source/includes/write/update.scala

```diff
@@ -0,0 +1,61 @@
+import org.mongodb.scala._
+import org.mongodb.scala.bson.Document
+import org.mongodb.scala.model.Filters.equal
+import org.mongodb.scala.model.UpdateOptions
+import org.mongodb.scala.model.Updates.{combine, rename, set}
+import org.mongodb.scala.result.UpdateResult
+
+object Update {
+
+  def main(args: Array[String]): Unit = {
+    val mongoClient = MongoClient("<connection string URI>")
+
+    // start-db-coll
+    val database: MongoDatabase = mongoClient.getDatabase("sample_restaurants")
+    val collection: MongoCollection[Document] = database.getCollection("restaurants")
+    // end-db-coll
+
+    // start-update-one
+    val filter = equal("name", "Happy Garden")
+    val update = set("name", "Mountain House")
+    val observable: Observable[UpdateResult] = collection.updateOne(filter, update)
+
+    observable.subscribe(new Observer[UpdateResult] {
+      override def onNext(result: UpdateResult): Unit = println(s"Updated document count: ${result.getModifiedCount}")
```

Comment:
S: in all examples, this line is causing scroll, consider newlining
```suggestion
      override def onNext(result: UpdateResult): Unit =
          println(s"Updated document count: ${result.getModifiedCount}")
```